components/raftstore/src/store/entry_storage.rs

```diff
@@ -0,0 +1,1414 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
Can this file be moved to raftstore/src/common/store instead so that it's straightforward that code is shared, as it's already moved anyway?
And I suppose there's no code change in this file comparing to the original peer_storage?

Reply:
Yes, no logic is changed.

Reply:
I will add a doc string to clarify it's shared as discussed.